entrypoint.sh

```diff
@@ -29,7 +29,7 @@ ROOT_OS_RELEASE="${ROOT_OS_RELEASE:-/root/etc/os-release}"
 KERNEL_SRC_DIR="${KERNEL_SRC_DIR:-/build/usr/src/linux}"
 NVIDIA_DRIVER_VERSION="${NVIDIA_DRIVER_VERSION:-390.46}"
 NVIDIA_DRIVER_MD5SUM="${NVIDIA_DRIVER_MD5SUM:-}"
-NVIDIA_DRIVER_DOWNLOAD_URL_DEFAULT="https://us.download.nvidia.com/tesla/${NVIDIA_DRIVER_VERSION}/NVIDIA-Linux-x86_64-${NVIDIA_DRIVER_VERSION}.run"
+NVIDIA_DRIVER_DOWNLOAD_URL_DEFAULT='https://storage.googleapis.com/nvidia-drivers-${NVIDIA_DRIVER_DOWNLOAD_LOCATION}-public/TESLA/NVIDIA-Linux-x86_64-'"${NVIDIA_DRIVER_VERSION}.run"
 NVIDIA_DRIVER_DOWNLOAD_URL="${NVIDIA_DRIVER_DOWNLOAD_URL:-$NVIDIA_DRIVER_DOWNLOAD_URL_DEFAULT}"
```

Comment:
I didn't set this to empty because NVIDIA_INSTALLER_RUNFILE requires this. Also, keep the default value at the beginning make it easier to change. Instead, I removed NVIDIA_DRIVER_DOWNLOAD_URL from here and added it below so that we can keep user's input as is.